### DISCUSSION TITLE
Plotted with opposite farm/pool public key!

### DISCUSSION THREAD

**Discussion by stubmirror** [Chia-Network/chia-blockchain#5610] [Q&A]
anyway to recover these plots plotted with the pool key set as public key and vice versa?
not a huge deal just some plots done on laptop but it explains why they never seemed to be farmed while harvesting

**Comment by mariano54** [comment]
if they were few plots, i would just replot them. its possible to farm them but will require some coding.

**Reply by stubmirror** [reply]
thanks if it requires some work dont worry about it, most of the damage is already done in that i didnt farm them for a week or 2 on the harvester, so ill just replot as iam mostly out of space anyways.
